ts/nni_manager/config/aml/amlUtil.py

```diff
@@ -52,7 +52,7 @@
                     print('stop_result:failed')
                     exit(0)
                 loop_count += 1
-                time.sleep(500)
+                time.sleep(5)
```

Comment:
`status` is never updated in loop body, so the result is deterministic............

Reply:
Oh it's in next line. My mistake.